.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     steps:
       - *attach_workspace
       - run: yarn build:demo
-      - run: scripts/deploy.js
+      - run: scripts/deploy.mjs
```

Comment:
Bummer the `mjs` extension is needed - maybe specifying `type` in `package.json` makes it work with `js`? 🤞🏻

Reply:
Important to note that the package `main` is `dist/index.svg`. We're not packaging an ESM module here. Again, referencing https://www.gatsbyjs.com/docs/how-to/custom-configuration/es-modules/, I _think_ using `.mjs` is the desired approach.

Reply:
Good call out. Thx for clarifying.